src/components/message-input/emoji-picker/emoji-picker.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { shallow } from 'enzyme';
 import React from 'react';
-import { ViewModes } from '../../shared-components/theme-engine';
+import { ViewModes } from '../../../shared-components/theme-engine';
```

Comment:
Love these little enhancements but next time can you submit them as a separate PR either before or after the actual changes.